### DISCUSSION TITLE
F-Droid

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-mobile#767] [Ideas]

**Comment by gmaclennan** [comment]
Copied from our Slack chat:

> FYI took a quick look at what is involved in publishing to F-Droid. Unfortunately, publishing to F-Droid requires that you let them sign the app (https://f-droid.org/docs/FAQ_-_App_Developers/?title=FAQ_-_App_Developers#what-about-signing) because they build the app from source themselves (to ensure it is open source). This means that if Mapeo is installed from F-Droid then p2p updates will not work (since the updates that we distribute will be signed with a different key). It also means that users can not switch from an F-Droid install to a regular APK install or a Google Play install without uninstalling (and loosing data) and then re-installing. This is a pretty frustrating limitation of installing from F-Droid. We could *only* work with the F-Droid signed version, but that would mean Google Play not working and existing APKs not being able to be updated.

[a comment by an automated account is not shown]
